src/common/schemas/PatchRetentionRule.yml

```
@@ -1,19 +1,29 @@
-type: object
-description: Updates to a rule to expire or retain data.
-properties:
-  type:
-    type: string
-    default: expire
-    enum:
-      - expire
-  everySeconds:
-    type: integer
-    format: int64
-    description: Duration in seconds for how long data will be kept in the database. 0 means infinite.
-    example: 86400
-    minimum: 0
-  shardGroupDurationSeconds:
-    type: integer
-    format: int64
-    description: Shard duration measured in seconds.
-required: [type]
+  type: object
+  properties:
+    type:
+      type: string
+      default: expire
+      enum:
+        - expire
+    everySeconds:
+      type: integer
+      format: int64
+      description: |
+        The duration in seconds for how long data will be kept in the database.
```

```suggestion
        The number of seconds to keep data.
```

Shouldn't use "duration" in this context b/c that's a different data type in InfluxDB - might be confusing.